integration_tests/service_b/src/schema.py

```diff
@@ -48,4 +55,16 @@ class Mutation(ObjectType):
     funny_mutation = FunnyMutation.Field()
 
 
-schema = build_schema(Query, Mutation, types=[FileNode, FunnyText, FileNodeAnother])
+class Query(ObjectType):
+    file = Field(lambda: FileNode)
+    multiple = Field(lambda: MultipleKeys)
```

Comment:
As I understand, there is no reason to make `multiple` as top level query.
Would be enough to add `MultipleKeys` in types